src/Kernel/Object.class.st

```diff
@@ -1353,6 +1353,14 @@ Object >> isSelfEvaluating [
 	^ self isLiteral
 ]
 
+{ #category : #'self evaluating' }
+Object >> isSelfEvaluating: alreadyVisited [
```

Comment:
If I understand correctly, this receiver of `isSelfEvaluating:` will be always a subclass of `Collection`. So, it would not make more sense to put this method in Collection instead of Object?

Reply:
Yes it looks like :)

Reply:
Thanks!